docs/lib/auth/fragments/flutter/hub_events/10_listen_events.md

```diff
@@ -1,6 +1,6 @@
 ```dart
-auth.events.startListening((hubEvent) {
-  switch(hubEvent["eventName"]) {
+StreamSubscription hubSubscription = Amplify.Hub.listen([HubChannel.Auth], (hubEvent) {
```

Comment:
if we are not showing what to do with `StreamSubscription hubSubscription` maybe we should remove it from the docs?

Reply:
I felt it was useful to show what this method actually returns.  This should be recognizable and meaningful for Flutter devs.

Reply:
Can we show one usage of this variable after the call?